.github/workflows/build_and_e2e.yml

```diff
@@ -1,10 +1,9 @@
 ---
-name: Build and push Docker Image
+name: Build Image and E2E
```

Comment:
Better name where Push workflow, also for file push.yml

Reply:
done